venia-integration-tests/src/fixtures/googleReCaptchaApi/reCaptchaV3ConfigDefault.json

```diff
@@ -5,6 +5,9 @@
             "badge_position": "bottomright",
             "language_code": "en",
             "forms": [
+                "CUSTOMER_FORGOT_PASSWORD",
+                "PLACE_ORDER",
+                "BRAINTREE"
```

Comment:
@jeremycharron missing `,` causing parsing error.

Reply:
Sorry about that. Fixed !